examples/analytics.amp.html

```diff
@@ -292,6 +292,55 @@
 </amp-analytics>
 <!-- End Mediametrie example -->
 
+
+<!-- mParticle tracking
+  Ensure you replace AppKey with your Application key
+-->
+<amp-analytics type="mparticle" id="mparticle">
+<script type="application/json">
+  {
+    "vars": {
+      "apiKey": "AppKey"
+    },
+    "triggers": {
+      "trackPageview": {
+        "on": "visible",
+        "request": "pageview"
+      },
+      "trackPageviewWithCustomTitleAndUrl": {
+        "on": "visible",
+        "request": "pageview",
+        "vars": {
+          "title": "Custom Title",
+          "ampdocUrl" : "http://www.example.com"
+        }
+      },
+      "trackAppEvent": {
+        "on": "click",
+        "selector": "#test1",
+        "request": "event",
+        "vars": {
+          "eventName": "AMP Test 1 button clicked",
+          "eventType": "Unknown",
+          "eventAttributes_Keys": ["test key1","test key2"],
```

Comment:
event attributes can be specified to pageview events also. They are optional for both pageview and event hits.